packages/system/src/components/homepage/content-banner.tsx

```diff
@@ -35,7 +35,7 @@ export function ContentBanner({
 	return (
 		<div className={classNames(className, contentBannerStyle)} {...props}>
 			<div className={contentBannerTextAreaStyle}>
-				<h2 className={contentBannerHeadingStyle}>Get smarter on the go</h2>
+				<h2 className={contentBannerHeadingStyle}>Discover new communities</h2>
```

Comment:
It looks like this change also affects the podcast section

<img width="685" alt="Screenshot 2023-03-02 at 9 25 31 AM" src="https://user-images.githubusercontent.com/67210629/222505072-d4768651-b26e-4c4b-af9d-55f92e1ebb59.png">